packages/dialtone-vue2/common/emoji/index.js

```diff
@@ -15,6 +15,8 @@ export let emojiFileExtensionLarge = '.png';
 
 export const emojiJson = emojiJsonLocal;
 
+export const emojiShortCodeRegex = /(^| ):\w+:/g;
```

Comment:
It's a pretty common case to put multiple emojis beside each other without spaces like so :smile::smile::smile::smile:. Maybe we can do a special case to not escape when there is a "double colon"

Reply:
Also if you use a unicode emoji in emoji-text-wrapper it allows you to do this, but not with shortnames

Reply:
> It's a pretty common case to put multiple emojis beside each other without spaces like so 😄😄😄😄. Maybe we can do a special case to not escape when there is a "double colon"

That implied the use of `Positive Lookbehind` which IIRC we just had an issue with it on older iOS devices.

Reply:
> Also if you use a unicode emoji in emoji-text-wrapper it allows you to do this, but not with shortnames

It's using the same emojiShortCodeRegex so this should be fixed too.

Reply:
> That implied the use of `Positive Lookbehind` which IIRC we just had an issue with it on older iOS devices.

hmm wasn't that negative lookbehind?

Reply:
> > That implied the use of `Positive Lookbehind` which IIRC we just had an issue with it on older iOS devices.
> 
> hmm wasn't that negative lookbehind?

Yeah, I think the issues were with negative lookbehind but they both have the same compatibility. Let's just keep an eye on this if some issues come related to it.